hivemind/server/__init__.py

```diff
@@ -70,9 +70,10 @@ def __init__(
 
     @staticmethod
     def create(listen_on='0.0.0.0:*', num_experts: int = None, expert_uids: str = None, expert_pattern: str = None,
-               expert_cls='ffn', hidden_dim=1024, optim_cls=torch.optim.Adam, num_handlers=None, max_batch_size=4096,
-               device=None, no_dht=False, initial_peers=(), dht_port=None, checkpoint_dir: Optional[Path] = None,
-               load_experts=False, compression=CompressionType.NONE, *, start: bool, **kwargs) -> Server:
+               expert_cls='ffn', hidden_dim=1024, optim_cls=torch.optim.Adam, scheduler: str = 'none',
+               num_warmup_steps=None, num_training_steps=None, num_handlers=None, max_batch_size=4096, device=None,
+               no_dht=False, initial_peers=(), dht_port=None, checkpoint_dir: Optional[Path] = None,
+               load_expert_states=False, compression=CompressionType.NONE, *, start: bool, **kwargs) -> Server:
```

Comment:
Can we check the necessity to load the expert only with the checkpoint_dir parameter?  If it,s None with initialize it from the scratch, if it's a path - we load state from this path.

Reply:
Done